Bump go to 1.20

siddarthkay opened the pull request:
## Summary

This PR attempts to upgrade go version to 1.20
This PR also removes the following items from lint checks : 
* `goconst`
* `structcheck`
* `deadcode`
* `golint`
* `varcheck`

Mobile PR for QA purposes -> https://github.com/status-im/status-mobile/pull/19564

## Todo

- [x] Store nodes not available on mobile -> https://github.com/status-im/status-mobile/issues/19581

apentori committed:
chore_: bump go to 1.20

This commit attempts to upgrade go version to 1.20.12
This commit also removes the following items from lint checks :
* `goconst`
* `structcheck`
* `deadcode`
* `golint`
* `varcheck`

Mobile PR for QA purposes -> https://github.com/status-im/status-mobile/pull/19564

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (115)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_multiplication_x: | 9a39cf1c7e817f223e7dfc468e778445d2f36068 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/1/) | 2024-04-08 17:59:32 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/1//consoleText) |
| :heavy_check_mark: | 9a39cf1c7e817f223e7dfc468e778445d2f36068 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/1/) | 2024-04-08 18:00:56 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240408-175649-9a39cf1c7e817f223e7dfc468e778445d2f36068-pr5027.zip) |
| :heavy_check_mark: | 9a39cf1c7e817f223e7dfc468e778445d2f36068 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/1/) | 2024-04-08 18:01:59 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240408-175649-9a39cf1c7e817f223e7dfc468e778445d2f36068-pr5027.zip) |
| :heavy_check_mark: | 9a39cf1c7e817f223e7dfc468e778445d2f36068 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/1/) | 2024-04-08 18:02:37 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240408-175644-9a39cf1c7e817f223e7dfc468e778445d2f36068-pr5027.aar) |
| | | | | | | |
| :heavy_check_mark: | 199eed7ce6d7517b980e552b130b6ca6da44e3b1 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/2/) | 2024-04-08 18:10:13 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240408-180829-199eed7ce6d7517b980e552b130b6ca6da44e3b1-pr5027.aar) |
| :heavy_check_mark: | 199eed7ce6d7517b980e552b130b6ca6da44e3b1 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/2/) | 2024-04-08 18:11:46 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240408-180829-199eed7ce6d7517b980e552b130b6ca6da44e3b1-pr5027.zip) |
| :heavy_check_mark: | 199eed7ce6d7517b980e552b130b6ca6da44e3b1 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/2/) | 2024-04-08 18:12:41 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240408-180833-199eed7ce6d7517b980e552b130b6ca6da44e3b1-pr5027.zip) |
| :heavy_check_mark: | 199eed7ce6d7517b980e552b130b6ca6da44e3b1 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/2/) | 2024-04-08 18:48:19 | ~39 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 3150f35b7a40576b59eb4e8328d16cce650f7cbe | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/3/) | 2024-04-09 07:13:59 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240409-071126-3150f35b7a40576b59eb4e8328d16cce650f7cbe-pr5027.zip) |
| :heavy_check_mark: | 3150f35b7a40576b59eb4e8328d16cce650f7cbe | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/3/) | 2024-04-09 07:14:36 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240409-071126-3150f35b7a40576b59eb4e8328d16cce650f7cbe-pr5027.aar) |
| :heavy_check_mark: | 3150f35b7a40576b59eb4e8328d16cce650f7cbe | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/3/) | 2024-04-09 07:15:15 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240409-071126-3150f35b7a40576b59eb4e8328d16cce650f7cbe-pr5027.zip) |
| :heavy_check_mark: | 3150f35b7a40576b59eb4e8328d16cce650f7cbe | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/3/) | 2024-04-09 07:51:27 | ~40 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | b2c7b72d425d6519e2d7694216823b644216b501 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/4/) | 2024-04-09 07:24:08 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240409-072230-b2c7b72d425d6519e2d7694216823b644216b501-pr5027.aar) |
| :heavy_check_mark: | b2c7b72d425d6519e2d7694216823b644216b501 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/4/) | 2024-04-09 07:24:48 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240409-072231-b2c7b72d425d6519e2d7694216823b644216b501-pr5027.zip) |
| :heavy_check_mark: | b2c7b72d425d6519e2d7694216823b644216b501 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/4/) | 2024-04-09 07:25:37 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240409-072231-b2c7b72d425d6519e2d7694216823b644216b501-pr5027.zip) |
| :heavy_check_mark: | b2c7b72d425d6519e2d7694216823b644216b501 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/4/) | 2024-04-09 08:30:24 | ~38 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 923d124a10b76c2d8ae715906cf997418bc920dc | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/5/) | 2024-04-09 09:17:53 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240409-091528-923d124a10b76c2d8ae715906cf997418bc920dc-pr5027.aar) |
| :heavy_check_mark: | 923d124a10b76c2d8ae715906cf997418bc920dc | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/5/) | 2024-04-09 09:18:00 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240409-091528-923d124a10b76c2d8ae715906cf997418bc920dc-pr5027.zip) |
| :heavy_check_mark: | 923d124a10b76c2d8ae715906cf997418bc920dc | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/5/) | 2024-04-09 09:19:33 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240409-091528-923d124a10b76c2d8ae715906cf997418bc920dc-pr5027.zip) |
| :heavy_check_mark: | 923d124a10b76c2d8ae715906cf997418bc920dc | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/5/) | 2024-04-09 09:58:02 | ~42 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 7fcf11b1646153906dce777abd41c734e1b7e76f | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/6/) | 2024-04-09 09:28:06 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240409-092543-7fcf11b1646153906dce777abd41c734e1b7e76f-pr5027.aar) |
| :heavy_check_mark: | 7fcf11b1646153906dce777abd41c734e1b7e76f | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/6/) | 2024-04-09 09:29:16 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240409-092543-7fcf11b1646153906dce777abd41c734e1b7e76f-pr5027.zip) |
| :heavy_check_mark: | 7fcf11b1646153906dce777abd41c734e1b7e76f | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/6/) | 2024-04-09 09:31:02 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240409-092543-7fcf11b1646153906dce777abd41c734e1b7e76f-pr5027.zip) |
| :heavy_check_mark: | 7fcf11b1646153906dce777abd41c734e1b7e76f | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/6/) | 2024-04-09 10:37:50 | ~39 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 6d28f90a799458f2394f16bbd5ef7eb833cb56f0 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/7/) | 2024-04-09 11:22:47 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240409-111939-6d28f90a799458f2394f16bbd5ef7eb833cb56f0-pr5027.aar) |
| :heavy_check_mark: | 6d28f90a799458f2394f16bbd5ef7eb833cb56f0 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/7/) | 2024-04-09 11:25:12 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240409-111943-6d28f90a799458f2394f16bbd5ef7eb833cb56f0-pr5027.zip) |
| :heavy_check_mark: | 6d28f90a799458f2394f16bbd5ef7eb833cb56f0 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/7/) | 2024-04-09 12:00:41 | ~40 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 3b8f754eabefe9868328b70d630dc7273e6b687c | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/8/) | 2024-04-09 11:43:48 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240409-114159-3b8f754eabefe9868328b70d630dc7273e6b687c-pr5027.aar) |
| :heavy_check_mark: | 3b8f754eabefe9868328b70d630dc7273e6b687c | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/8/) | 2024-04-09 11:45:40 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240409-114203-3b8f754eabefe9868328b70d630dc7273e6b687c-pr5027.zip) |
| :heavy_check_mark: | 3b8f754eabefe9868328b70d630dc7273e6b687c | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/8/) | 2024-04-09 12:38:50 | ~37 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/8//consoleText) |
| | | | | | | |
| :heavy_check_mark: | ddff9e076d0d1787af2aaa7842574d9ebbcfbae0 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/9/) | 2024-04-09 15:18:54 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240409-151631-ddff9e076d0d1787af2aaa7842574d9ebbcfbae0-pr5027.aar) |
| :heavy_check_mark: | ddff9e076d0d1787af2aaa7842574d9ebbcfbae0 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/9/) | 2024-04-09 15:19:11 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240409-151635-ddff9e076d0d1787af2aaa7842574d9ebbcfbae0-pr5027.zip) |
| :heavy_check_mark: | ddff9e076d0d1787af2aaa7842574d9ebbcfbae0 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/9/) | 2024-04-09 15:23:26 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240409-151631-ddff9e076d0d1787af2aaa7842574d9ebbcfbae0-pr5027.zip) |
| :heavy_check_mark: | ddff9e076d0d1787af2aaa7842574d9ebbcfbae0 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/9/) | 2024-04-09 15:56:03 | ~39 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/9//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | caaa3edf3a2a5781e9f26bcc3a27d28cd3c8ca9e | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/10/) | 2024-04-11 06:02:54 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/10//consoleText) |
| :heavy_check_mark: | caaa3edf3a2a5781e9f26bcc3a27d28cd3c8ca9e | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/10/) | 2024-04-11 06:04:29 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240411-060130-caaa3edf3a2a5781e9f26bcc3a27d28cd3c8ca9e-pr5027.zip) |
| :heavy_check_mark: | caaa3edf3a2a5781e9f26bcc3a27d28cd3c8ca9e | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/10/) | 2024-04-11 06:05:44 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240411-060126-caaa3edf3a2a5781e9f26bcc3a27d28cd3c8ca9e-pr5027.aar) |
| :heavy_check_mark: | caaa3edf3a2a5781e9f26bcc3a27d28cd3c8ca9e | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/10/) | 2024-04-11 06:06:34 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240411-060130-caaa3edf3a2a5781e9f26bcc3a27d28cd3c8ca9e-pr5027.zip) |
| | | | | | | |
| :heavy_check_mark: | 88c8b2e0aed69f3ee92c10556ba1a65381b167c3 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/11/) | 2024-04-11 06:07:45 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240411-060501-88c8b2e0aed69f3ee92c10556ba1a65381b167c3-pr5027.zip) |
| :heavy_check_mark: | 88c8b2e0aed69f3ee92c10556ba1a65381b167c3 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/11/) | 2024-04-11 06:08:03 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240411-060603-88c8b2e0aed69f3ee92c10556ba1a65381b167c3-pr5027.aar) |
| :heavy_check_mark: | 88c8b2e0aed69f3ee92c10556ba1a65381b167c3 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/11/) | 2024-04-11 06:10:11 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240411-060700-88c8b2e0aed69f3ee92c10556ba1a65381b167c3-pr5027.zip) |
| :heavy_check_mark: | 88c8b2e0aed69f3ee92c10556ba1a65381b167c3 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/12/) | 2024-04-11 07:23:36 | ~39 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/12//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a474965c6b93ce8acb74a44de9b3b1c896dcb8b9 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/11/) | 2024-04-11 06:44:12 | ~40 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/11//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 0e4620243b4351df6520d66af1ce265efc0d8b1e | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/12/) | 2024-04-12 07:33:06 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240412-073042-0e4620243b4351df6520d66af1ce265efc0d8b1e-pr5027.zip) |
| :heavy_check_mark: | 0e4620243b4351df6520d66af1ce265efc0d8b1e | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/12/) | 2024-04-12 07:33:13 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240412-073038-0e4620243b4351df6520d66af1ce265efc0d8b1e-pr5027.aar) |
| :heavy_check_mark: | 0e4620243b4351df6520d66af1ce265efc0d8b1e | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/12/) | 2024-04-12 07:34:29 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240412-073042-0e4620243b4351df6520d66af1ce265efc0d8b1e-pr5027.zip) |
| :heavy_check_mark: | 0e4620243b4351df6520d66af1ce265efc0d8b1e | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/13/) | 2024-04-12 08:10:58 | ~40 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/13//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 3288d389915ed03f04d8d1e03cc15d83198ab913 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/14/) | 2024-04-13 11:25:29 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/14//consoleText) |
| :heavy_check_mark: | 3288d389915ed03f04d8d1e03cc15d83198ab913 | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/13/) | 2024-04-13 11:26:06 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240413-112347-3288d389915ed03f04d8d1e03cc15d83198ab913-pr5027.aar) |
| :heavy_check_mark: | 3288d389915ed03f04d8d1e03cc15d83198ab913 | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/13/) | 2024-04-13 11:26:16 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240413-112352-3288d389915ed03f04d8d1e03cc15d83198ab913-pr5027.zip) |
| :heavy_check_mark: | 3288d389915ed03f04d8d1e03cc15d83198ab913 | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/13/) | 2024-04-13 11:27:01 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240413-112352-3288d389915ed03f04d8d1e03cc15d83198ab913-pr5027.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 70603a047ef2ba129431a2ae821d14b230f5a508 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/15/) | 2024-04-16 10:33:12 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/15//consoleText) |
| :heavy_check_mark: | 70603a047ef2ba129431a2ae821d14b230f5a508 | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/14/) | 2024-04-16 10:33:20 | ~1 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240416-103145-70603a047ef2ba129431a2ae821d14b230f5a508-pr5027.zip) |
| :heavy_check_mark: | 70603a047ef2ba129431a2ae821d14b230f5a508 | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/14/) | 2024-04-16 10:35:44 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240416-103145-70603a047ef2ba129431a2ae821d14b230f5a508-pr5027.zip) |
| :heavy_check_mark: | 70603a047ef2ba129431a2ae821d14b230f5a508 | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/14/) | 2024-04-16 10:36:32 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240416-103145-70603a047ef2ba129431a2ae821d14b230f5a508-pr5027.aar) |
| | | | | | | |
| :heavy_multiplication_x: | 0212f93dd55bd9266cea62fb56a2b6a86777e8cd | [#16](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/16/) | 2024-04-16 10:34:09 | ~44 sec | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/16//consoleText) |
| :heavy_check_mark: | 0212f93dd55bd9266cea62fb56a2b6a86777e8cd | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/15/) | 2024-04-16 10:35:30 | ~1 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240416-103341-0212f93dd55bd9266cea62fb56a2b6a86777e8cd-pr5027.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 473bb92bdaf1cc4e7a454160e709fdee8f6aa9d1 | [#17](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/17/) | 2024-04-16 10:35:28 | ~42 sec | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/17//consoleText) |
| :heavy_check_mark: | 473bb92bdaf1cc4e7a454160e709fdee8f6aa9d1 | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/15/) | 2024-04-16 10:38:49 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240416-103601-473bb92bdaf1cc4e7a454160e709fdee8f6aa9d1-pr5027.zip) |
| :heavy_check_mark: | 473bb92bdaf1cc4e7a454160e709fdee8f6aa9d1 | [#16](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/16/) | 2024-04-16 10:39:53 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240416-103556-473bb92bdaf1cc4e7a454160e709fdee8f6aa9d1-pr5027.zip) |
| :heavy_check_mark: | 473bb92bdaf1cc4e7a454160e709fdee8f6aa9d1 | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/15/) | 2024-04-16 10:42:34 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240416-103648-473bb92bdaf1cc4e7a454160e709fdee8f6aa9d1-pr5027.aar) |
| | | | | | | |
| :heavy_multiplication_x: | cc8991092700a862a953a1c3f42e703cf1220ec3 | [#18](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/18/) | 2024-04-16 10:38:36 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/18//consoleText) |
| :heavy_check_mark: | cc8991092700a862a953a1c3f42e703cf1220ec3 | [#16](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/16/) | 2024-04-16 10:41:42 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240416-103906-cc8991092700a862a953a1c3f42e703cf1220ec3-pr5027.zip) |
| :heavy_check_mark: | cc8991092700a862a953a1c3f42e703cf1220ec3 | [#17](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/17/) | 2024-04-16 10:43:47 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240416-104015-cc8991092700a862a953a1c3f42e703cf1220ec3-pr5027.zip) |
| | | | | | | |
| :heavy_check_mark: | 3f3bd6c59fda8895d95e44dd05cf5bd9d1b034ea | [#17](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/17/) | 2024-04-16 10:44:21 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240416-104201-3f3bd6c59fda8895d95e44dd05cf5bd9d1b034ea-pr5027.zip) |
| :heavy_check_mark: | 3f3bd6c59fda8895d95e44dd05cf5bd9d1b034ea | [#16](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/16/) | 2024-04-16 10:45:39 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240416-104253-3f3bd6c59fda8895d95e44dd05cf5bd9d1b034ea-pr5027.aar) |
| :heavy_check_mark: | 3f3bd6c59fda8895d95e44dd05cf5bd9d1b034ea | [#18](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/18/) | 2024-04-16 10:47:28 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240416-104410-3f3bd6c59fda8895d95e44dd05cf5bd9d1b034ea-pr5027.zip) |
| :heavy_multiplication_x: | 3f3bd6c59fda8895d95e44dd05cf5bd9d1b034ea | [#19](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/19/) | 2024-04-16 10:47:40 | ~6 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/19//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 2695e6194237ec60c1406eb032f9719fd59c1139 | [#17](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/17/) | 2024-04-16 10:54:11 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240416-105230-2695e6194237ec60c1406eb032f9719fd59c1139-pr5027.aar) |
| :heavy_check_mark: | 2695e6194237ec60c1406eb032f9719fd59c1139 | [#18](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/18/) | 2024-04-16 10:54:55 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240416-105230-2695e6194237ec60c1406eb032f9719fd59c1139-pr5027.zip) |
| :heavy_check_mark: | 2695e6194237ec60c1406eb032f9719fd59c1139 | [#19](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/19/) | 2024-04-16 10:56:30 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240416-105230-2695e6194237ec60c1406eb032f9719fd59c1139-pr5027.zip) |
| :heavy_multiplication_x: | 2695e6194237ec60c1406eb032f9719fd59c1139 | [#20](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/20/) | 2024-04-16 10:58:35 | ~6 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/20//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c7bbecde9b9bb5e764f0774200b2358d0aeabceb | [#18](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/18/) | 2024-04-16 11:07:13 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240416-110450-c7bbecde9b9bb5e764f0774200b2358d0aeabceb-pr5027.aar) |
| :heavy_check_mark: | c7bbecde9b9bb5e764f0774200b2358d0aeabceb | [#19](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/19/) | 2024-04-16 11:07:34 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240416-110454-c7bbecde9b9bb5e764f0774200b2358d0aeabceb-pr5027.zip) |
| :heavy_check_mark: | c7bbecde9b9bb5e764f0774200b2358d0aeabceb | [#20](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/20/) | 2024-04-16 11:08:38 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240416-110450-c7bbecde9b9bb5e764f0774200b2358d0aeabceb-pr5027.zip) |
| :heavy_check_mark: | c7bbecde9b9bb5e764f0774200b2358d0aeabceb | [#21](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/21/) | 2024-04-16 11:43:42 | ~38 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/21//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 4ca3435d2737fdb9977077e6854c2325a2383ec7 | [#20](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/20/) | 2024-04-17 14:55:57 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240417-145311-4ca3435d2737fdb9977077e6854c2325a2383ec7-pr5027.zip) |
| :heavy_check_mark: | 4ca3435d2737fdb9977077e6854c2325a2383ec7 | [#19](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/19/) | 2024-04-17 14:56:12 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240417-145311-4ca3435d2737fdb9977077e6854c2325a2383ec7-pr5027.aar) |
| :heavy_check_mark: | 4ca3435d2737fdb9977077e6854c2325a2383ec7 | [#21](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/21/) | 2024-04-17 14:57:04 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240417-145311-4ca3435d2737fdb9977077e6854c2325a2383ec7-pr5027.zip) |
| :heavy_check_mark: | 4ca3435d2737fdb9977077e6854c2325a2383ec7 | [#22](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/22/) | 2024-04-17 15:33:27 | ~40 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/22//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 6dfb180a36de53d84980b69f89febdcca9140ef3 | [#21](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/21/) | 2024-04-17 14:58:43 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240417-145613-6dfb180a36de53d84980b69f89febdcca9140ef3-pr5027.zip) |
| :heavy_check_mark: | 6dfb180a36de53d84980b69f89febdcca9140ef3 | [#20](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/20/) | 2024-04-17 14:58:46 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240417-145628-6dfb180a36de53d84980b69f89febdcca9140ef3-pr5027.aar) |
| | | | | | | |
| :heavy_check_mark: | cc575867ab0233b2dc81908b1b25b448a4901edf | [#21](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/21/) | 2024-04-17 15:00:54 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240417-145903-cc575867ab0233b2dc81908b1b25b448a4901edf-pr5027.aar) |
| :heavy_check_mark: | cc575867ab0233b2dc81908b1b25b448a4901edf | [#22](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/22/) | 2024-04-17 15:01:09 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240417-145901-cc575867ab0233b2dc81908b1b25b448a4901edf-pr5027.zip) |
| :heavy_check_mark: | cc575867ab0233b2dc81908b1b25b448a4901edf | [#22](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/22/) | 2024-04-17 15:01:29 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240417-145754-cc575867ab0233b2dc81908b1b25b448a4901edf-pr5027.zip) |
| :heavy_check_mark: | cc575867ab0233b2dc81908b1b25b448a4901edf | [#23](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/23/) | 2024-04-17 15:05:13 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240417-150149-cc575867ab0233b2dc81908b1b25b448a4901edf-pr5027.zip) |
| :heavy_check_mark: | cc575867ab0233b2dc81908b1b25b448a4901edf | [#23](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/23/) | 2024-04-17 16:13:33 | ~39 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/23//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 0b81ffdfc16cf4f278e68661f2ae46dcc1baa6a7 | [#22](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/22/) | 2024-04-18 07:51:15 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-074907-0b81ffdfc16cf4f278e68661f2ae46dcc1baa6a7-pr5027.aar) |
| :heavy_check_mark: | 0b81ffdfc16cf4f278e68661f2ae46dcc1baa6a7 | [#23](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/23/) | 2024-04-18 07:51:38 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240418-074911-0b81ffdfc16cf4f278e68661f2ae46dcc1baa6a7-pr5027.zip) |
| :heavy_check_mark: | 0b81ffdfc16cf4f278e68661f2ae46dcc1baa6a7 | [#24](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/24/) | 2024-04-18 07:52:30 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240418-074907-0b81ffdfc16cf4f278e68661f2ae46dcc1baa6a7-pr5027.zip) |
| :heavy_check_mark: | 0b81ffdfc16cf4f278e68661f2ae46dcc1baa6a7 | [#24](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/24/) | 2024-04-18 08:29:05 | ~39 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/24//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 71d8aa57e555d65edb4226b60b817a8b11467523 | [#23](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/23/) | 2024-04-18 08:08:57 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-080657-71d8aa57e555d65edb4226b60b817a8b11467523-pr5027.aar) |
| :heavy_check_mark: | 71d8aa57e555d65edb4226b60b817a8b11467523 | [#24](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/24/) | 2024-04-18 08:09:25 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240418-080657-71d8aa57e555d65edb4226b60b817a8b11467523-pr5027.zip) |
| :heavy_check_mark: | 71d8aa57e555d65edb4226b60b817a8b11467523 | [#25](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/25/) | 2024-04-18 08:14:05 | ~7 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240418-080657-71d8aa57e555d65edb4226b60b817a8b11467523-pr5027.zip) |
| | | | | | | |
| :heavy_check_mark: | 06096b8decbd65467a92a7c77aa2ed3b70972aa8 | [#24](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/24/) | 2024-04-18 08:28:52 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-082642-06096b8decbd65467a92a7c77aa2ed3b70972aa8-pr5027.aar) |
| :heavy_check_mark: | 06096b8decbd65467a92a7c77aa2ed3b70972aa8 | [#25](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/25/) | 2024-04-18 08:29:42 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240418-082646-06096b8decbd65467a92a7c77aa2ed3b70972aa8-pr5027.zip) |
| :heavy_check_mark: | 06096b8decbd65467a92a7c77aa2ed3b70972aa8 | [#26](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/26/) | 2024-04-18 08:30:47 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240418-082646-06096b8decbd65467a92a7c77aa2ed3b70972aa8-pr5027.zip) |
| :heavy_check_mark: | 06096b8decbd65467a92a7c77aa2ed3b70972aa8 | [#25](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/25/) | 2024-04-18 09:09:49 | ~40 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/25//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 7955d76b7bd11a6716a698dd39d6473e8fadd626 | [#25](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/25/) | 2024-04-18 09:01:53 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-085928-7955d76b7bd11a6716a698dd39d6473e8fadd626-pr5027.aar) |
| | | | | | | |
| :heavy_check_mark: | 94c19e80152e41a80baf3dec40ff09c9c779f4d1 | [#27](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/27/) | 2024-04-18 09:02:22 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240418-085948-94c19e80152e41a80baf3dec40ff09c9c779f4d1-pr5027.zip) |
| :heavy_check_mark: | 94c19e80152e41a80baf3dec40ff09c9c779f4d1 | [#26](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/26/) | 2024-04-18 09:06:21 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-090220-94c19e80152e41a80baf3dec40ff09c9c779f4d1-pr5027.aar) |
| :heavy_check_mark: | 94c19e80152e41a80baf3dec40ff09c9c779f4d1 | [#28](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/28/) | 2024-04-18 09:07:49 | ~8 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240418-085948-94c19e80152e41a80baf3dec40ff09c9c779f4d1-pr5027.zip) |
| :heavy_multiplication_x: | 94c19e80152e41a80baf3dec40ff09c9c779f4d1 | [#26](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/26/) | 2024-04-18 09:15:28 | ~5 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/26//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e13349501798f7eb16aeec9aba9de8ac3c617314 | [#28](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/28/) | 2024-04-18 11:06:37 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240418-110344-e13349501798f7eb16aeec9aba9de8ac3c617314-pr5027.zip) |
| :heavy_check_mark: | e13349501798f7eb16aeec9aba9de8ac3c617314 | [#27](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/27/) | 2024-04-18 11:06:59 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-110340-e13349501798f7eb16aeec9aba9de8ac3c617314-pr5027.aar) |
| :heavy_check_mark: | e13349501798f7eb16aeec9aba9de8ac3c617314 | [#29](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/29/) | 2024-04-18 11:07:39 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240418-110344-e13349501798f7eb16aeec9aba9de8ac3c617314-pr5027.zip) |
| :heavy_check_mark: | e13349501798f7eb16aeec9aba9de8ac3c617314 | [#27](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/27/) | 2024-04-18 11:46:24 | ~42 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/27//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 3255da8c5c343145488df099dfa41e4f6434fa97 | [#28](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/28/) | 2024-04-18 12:16:56 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-121431-3255da8c5c343145488df099dfa41e4f6434fa97-pr5027.aar) |
| :heavy_check_mark: | 3255da8c5c343145488df099dfa41e4f6434fa97 | [#29](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/29/) | 2024-04-18 12:17:07 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240418-121431-3255da8c5c343145488df099dfa41e4f6434fa97-pr5027.zip) |
| :heavy_check_mark: | 3255da8c5c343145488df099dfa41e4f6434fa97 | [#30](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/30/) | 2024-04-18 12:18:04 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240418-121431-3255da8c5c343145488df099dfa41e4f6434fa97-pr5027.zip) |
| :heavy_check_mark: | 3255da8c5c343145488df099dfa41e4f6434fa97 | [#28](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/28/) | 2024-04-18 12:55:29 | ~40 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/28//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 7a1af109ebf045a84692401b18e7bd3dc2218255 | [#29](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/29/) | 2024-04-18 14:43:06 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-144038-7a1af109ebf045a84692401b18e7bd3dc2218255-pr5027.aar) |
| :heavy_check_mark: | 7a1af109ebf045a84692401b18e7bd3dc2218255 | [#30](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/30/) | 2024-04-18 14:43:16 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240418-144038-7a1af109ebf045a84692401b18e7bd3dc2218255-pr5027.zip) |
| :heavy_check_mark: | 7a1af109ebf045a84692401b18e7bd3dc2218255 | [#31](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/31/) | 2024-04-18 14:56:10 | ~15 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240418-144038-7a1af109ebf045a84692401b18e7bd3dc2218255-pr5027.zip) |
| :heavy_check_mark: | 7a1af109ebf045a84692401b18e7bd3dc2218255 | [#29](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/29/) | 2024-04-18 15:23:06 | ~42 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/29//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | ef96e22f4508bad74be77c79889e430d0118e8f4 | [#32](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/32/) | 2024-04-18 15:11:09 | ~59 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/32/consoleText) |
| :heavy_check_mark: | ef96e22f4508bad74be77c79889e430d0118e8f4 | [#31](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/31/) | 2024-04-18 15:12:34 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240418-151008-ef96e22f4508bad74be77c79889e430d0118e8f4-pr5027.zip) |
| :heavy_check_mark: | ef96e22f4508bad74be77c79889e430d0118e8f4 | [#30](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/30/) | 2024-04-18 15:12:38 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-151008-ef96e22f4508bad74be77c79889e430d0118e8f4-pr5027.aar) |
| :heavy_check_mark: | ef96e22f4508bad74be77c79889e430d0118e8f4 | [#30](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/30/) | 2024-04-18 16:03:19 | ~39 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/30//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f5215307ebc275c121e127ee22b9f6bcb737e958 | [#32](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5027/32/) | 2024-04-18 16:01:44 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240418-155923-f5215307ebc275c121e127ee22b9f6bcb737e958-pr5027.zip) |
| :heavy_check_mark: | f5215307ebc275c121e127ee22b9f6bcb737e958 | [#31](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5027/31/) | 2024-04-18 16:01:51 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240418-155919-f5215307ebc275c121e127ee22b9f6bcb737e958-pr5027.aar) |
| :heavy_check_mark: | f5215307ebc275c121e127ee22b9f6bcb737e958 | [#33](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5027/33/) | 2024-04-18 16:03:06 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240418-155923-f5215307ebc275c121e127ee22b9f6bcb737e958-pr5027.zip) |
| :heavy_check_mark: | f5215307ebc275c121e127ee22b9f6bcb737e958 | [#31](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/31/) | 2024-04-18 16:43:20 | ~39 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5027/31//consoleText) |

Samyoul commented:
Hey @siddarthkay! Thank you for your PR and your great work upgrading basically everything. Just to give my opinion, I'm uneasy about dropping `gosec`.

Other two linter checks I'm not too bothered about, they do help keep the code idiomatic though, particularly `structcheck`, it enforces struct usage in the same way the compiler treats var and import usage. I'm least attached to `goconst`, I don't mind repeated string literals.

siddarthkay commented:
`make lint` gives these warnings so it seems safe to remove these from enabled section of `.golangci.yml`

```
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have 
abandoned the linter. Replaced by unused.
WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has 
been archived by the owner. Replaced by revive.
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have 
abandoned the linter. Replaced by unused.
```

siddarthkay commented:
I enabled `gosec` to see the offending code and to fix them and following are the issues : 

```
protocol/communities/persistence.go:963:11: G202: SQL string concatenation (gosec)
	query := "SELECT sig, timestamp, topic, payload, padding, hash, third_party_id FROM 
	waku_messages WHERE timestamp >= " + fmt.Sprint(from) + " AND timestamp < " + fmt.Sprint(to) +
	 " AND ("
	         ^
transactions/testhelpers.go:147:12: G601: Implicit memory aliasing in for loop. (gosec)
					tx := &sum.tx
					      ^
transactions/testhelpers.go:169:12: G601: Implicit memory aliasing in for loop. (gosec)
					tx := &sum.tx
					      ^
transactions/transactor_test.go:303:12: G601: Implicit memory aliasing in for loop. (gosec)
				Return(&scenario.nonceFromNetwork, nil)
				       ^
services/wallet/transfer/block_ranges_sequential_dao.go:118:11: G202: SQL string concatenation (gosec)
	query := "SELECT address, blk_start, blk_first, blk_last, token_blk_start, token_blk_first, 
	token_blk_last, balance_check_hash FROM blocks_ranges_sequential WHERE address IN (" +
		addressesPlaceholder + ") AND network_id = ?"
		
protocol/activity_center_persistence.go:856:12: G202: SQL string concatenation (gosec)
	update := "UPDATE activity_center_notifications SET deleted = 1, updated_at = ? WHERE id IN (" + 
	inVector + ") AND NOT deleted"
	          ^
multiaccounts/accounts/keycard_database.go:266:11: G202: SQL string concatenation (gosec)
	query := `
		DELETE
		FROM
			keycards_accounts
		WHERE
			keycard_uid = ?
		AND
			account_address	IN (?` + inVector + `)`
			
protocol/message_persistence.go:3019:9: G202: SQL string concatenation (gosec)
	sql := "UPDATE user_messages SET response_to = ? WHERE id IN 
	(?" + strings.Repeat(",?", len(statusMessagesToUpdate)-1) + ")"
	       ^
	       
make: *** [lint] Error 1
```

The key offenders are : 
- G202: SQL string concatenation
- G601: Implicit memory aliasing in for loop.

siddarthkay commented:
Regarding G202: SQL string concatenation -> https://securego.io/docs/rules/g201-g202#the-right-way
It seems that rule exists to avoid SQL injection.
Which is nice, However we rarely take in any input from the user to generate these SQL statements so it seems safe to skip the rule in places by adding a  `// nolint: gosec`
I do see presence of `// nolint: gosec` in many places .. 121 times :D

siddarthkay commented:
next up is `G601: Implicit memory aliasing in for loop`

An easy fix for such cases is defining a local variable for memory references in for loop and use that variable ahead.
Only drawback of such an approach is weird looking code like this -> https://github.com/status-im/status-go/pull/5027/commits/3150f35b7a40576b59eb4e8328d16cce650f7cbe

We could also exclude just `G601` and still keep other rules of `gosec`.

which approach do you prefer @Samyoul ?

stefandunca commented:
> ...
> We could also exclude just `G601` and still keep other rules of `gosec`.
> 
There were few ugly bugs due to ignoring this check, mainly trying to modify a temporary variable instead of the original data. I think it is better to keep the check and fix the aliasing using temporary variables or index until we upgrade to 1.22 when this "optimization" is finally fixed

siddarthkay commented:
Thanks for weighing in @stefandunca, just to confirm, the changes in this commit -> https://github.com/status-im/status-go/pull/5027/commits/3150f35b7a40576b59eb4e8328d16cce650f7cbe seem good to you ?

stefandunca commented:
> Thanks for weighing in @stefandunca, just to confirm, the changes in this commit -> [3150f35](https://github.com/status-im/status-go/commit/3150f35b7a40576b59eb4e8328d16cce650f7cbe) seem good to you ?

They seems right if you need a quick fix and don't have time to look if the G202 can be fixed instead of ignoring. I added some comments for minor improvements and to add a follow up tasks for G202. However, please note that Run from tests can be parallelized and the aliasing become a real issue, therefore the G601 being a serious thing.

siddarthkay commented:
Thanks @stefandunca : reversing the alias is a good idea, I'll push a commit and fix that.
Regarding `G202: SQL string concatenation` : we could modify those SQL queries to make sure we don't generate them via string concatenation via `+` or `fmt.sprintf` and instead pass the correct parameters as suggested in https://securego.io/docs/rules/g201-g202#the-right-way 
I'll make a new issue for it, as I believe its better to do in a separate PR and would require some extensive QA since modifying queries could potentially break certain functionalities if we miss something.

siddarthkay commented:
issue created to track `fixing G202: SQL string concatenation` -> https://github.com/status-im/status-go/issues/5033

igor-sirotin reviewed (approved):
Thank you! 🎉

caybro commented:
Out of curiosity, why don't we bump to 1.21? :) See the issue I reported half a year ago: https://github.com/status-im/status-go/issues/4299

siddarthkay commented:
@caybro : because we observed an issue when upgrading to 1.20 https://github.com/status-im/status-mobile/issues/19581 and we want to fix that before we upgrade 2 major versions above the current one.
1.21 upgrade is next in line once we fix this issue...

siddarthkay commented:
converting to Draft because of storenode madness

siddarthkay commented:
@igor-sirotin : don't worry we found a dumb fix for the storenode problem  :)

cammellos commented on vendor/modules.txt:
```diff
@@ -927,7 +927,7 @@ github.com/status-im/go-multiaddr-ethv4
 ## explicit
 github.com/status-im/keycard-go/derivationpath
-# github.com/status-im/markdown v0.0.0-20240404192634-b7e33c6ac3d4
+# github.com/status-im/markdown v0.0.0-20231114210825-6c2d15b5dc57
```
why this has been changed? could you check that is pointing to the right version and not downgrading?

siddarthkay replied:
Apologies for the mistake! I've restored this change :)

jakubgs commented on protocol/messenger_mailserver_cycle.go:
```diff
@@ -27,6 +28,8 @@ const defaultBackoff = 10 * time.Second
 const isAndroidEmulator = runtime.GOOS == "android" && runtime.GOARCH == "amd64"
 const findNearestMailServer = !isAndroidEmulator
+const overrideDNS = runtime.GOOS == "android"
+const bootstrapDNS = "8.8.8.8:53"
```
I don't like this. We should at least leave a `FIXME` comment explaining that this should be changed.

It's a bard practice because:

- It's bad in terms of preserving privacy of our users.
- It doesn't respect user DNS settings on device or in their local network.
- It unreliable since it depends on only on server.

siddarthkay replied:
Ah sorry I missed adding the FIXME.. there is an issue open for this DNS entry, we also do the same thing for waku nodes in mobile config -> https://github.com/status-im/status-go/issues/3024

jakubgs reviewed (commented):
Good work.
